thoth/messaging/inspection_complete.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# thoth-messaging
+# Copyright(C) 2019, 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth Messaging module for InspectionCompleteMessage."""
+
+from .message_base import MessageBase, BaseMessageContents
+
+
+class InspectionCompletedMessage(MessageBase):
+    """Class used to indicate when amun inspections complete."""
+
+    topic_name = "thoth.inspection-completed"
+    _message_version = 1  # update on schema change
+
+    class MessageContents(BaseMessageContents, serializer="json"):  # type: ignore
+        """Class used to represent contents of a inspection-completed message Kafka topic."""
+
+        inspection_id: str
```

Comment:
I wonder if we should add also Amun API URL and deployment name from the configmap, so we can track down the cluster:

https://github.com/thoth-station/thoth-application/blob/4ca617b49cead4cd6650cdda39fc562b94e7c3ae/amun/overlays/amun-api/configmaps.yaml#L7-L8

Reply:
Isn't that part of an inspection?

Reply:
It is. If we want to obtain the inspection result based on `inspection_id` on Thoth side, we need to know which Amun API/deployment should be reached.

Reply:
chk